include/boost/beast/core/detail/config.hpp

```diff
@@ -113,4 +113,8 @@ namespace net = boost::asio;
 
 #endif
 
+#ifndef BOOST_BEAST_FILE_BUFFER_SIZE
+#define BOOST_BEAST_FILE_BUFFER_SIZE 4096
```

Comment:
This is more ugly than my ex but I guess its better than the alternative, which is nothing. That number should probably be higher, no?

Reply:
4096 is the previous default and it's asio's default, too.